libs/datasets/combined_datasets.py

```diff
@@ -111,7 +111,9 @@
 )
 
 
-def build_us_with_all_fields() -> Tuple[TimeseriesDataset, LatestValuesDataset]:
+def build_us_with_all_fields() -> Tuple[
```

Comment:
The meaning of this function feels like it's drifted - this is a bit verbose, but maybe this could be named something along the lines of  `load_data_sources_and_build_us_combined_datasets()`?

Reply:
Agreed that it has drifted and not so great as a function now. I took a different approach with 5ac9081 and copied it to the one place calling it.